### ISSUE TITLE
Mark Auto-Attend

### ISSUE THREAD

**Issue by ryanleesipes** [GetTogetherComm/GetTogether#138]
I would like to be able to set myself to attend all meetings on a team by default unless I mark otherwise.

**Comment by mhall119** [comment]
You mean for meetings that you didn't create?

**Comment by ryanleesipes** [comment]
Yeah, I've even been added as an administrator for [this one](https://gettogether.community/denver-open-source-remoters-club/), and I can't find how to auto-attend.

**Comment by mhall119** [comment]
There is no auto-attend feature yet, but now it's on the list :)
